tests/integrational/native_threads/test_file_upload.py

```diff
@@ -36,7 +36,7 @@ def callback(self, response, status):
         "tests/integrational/fixtures/native_threads/file_upload/send_file.yaml",
         filter_query_parameters=('pnsdk',)
     )
-    def test_send_file(self):
```

Comment:
this method was incorrectly treated as a test, not as a convenience method and was throwing warnings.